smpplib/client.py

```diff
@@ -176,6 +179,10 @@ def send_pdu(self, p):
                 consts.DESCRIPTIONS[consts.SMPP_ESME_RINVBNDSTS],
             ))
 
+        if self.max_outstanding_operations and p.command[-5:] != '_resp':
```

Comment:
It's better to use `pdu.is_request()` and `pdu.is_response()` to tell if it's a response

Reply:
Didn't notice those functions it'll be better indeed.